lib/cli/index.js

```diff
@@ -15,37 +15,37 @@ program.option('--stack-limit <limit>', 'change the stack trace limit', parseInt
 // tasks
 
 program
-	.command('show-client-config')
+	.command('show:client-config')
```

Comment:
Also it feels like this command requires an app, why not make it an argument like `show:config`?